Lesson1_business-card/index.html

```diff
@@ -17,27 +17,27 @@
         <div class="personal-data">
           <div class="name-data">
             <div class="name-surname">
-              <div class="text name">Aleksander</div>
-              <div class="text surname">KOWALSKI</div>
+              <div class="name">Aleksander</div>
+              <div class="surname">KOWALSKI</div>
             </div>
-            <div class="text work">STANOWISKO</div>
+            <div class="work">STANOWISKO</div>
           </div>
-          <div class="text contact-data">
+          <div class="contact-data">
             <div class="street">
                 <img src="map-pin.svg">
-              <span class="street_txt"> 
+              <span class="street-txt"> 
                 ul.Długa 6, Gdańsk
               </span>
             </div>
             <div class="email">
               <img src="mail.svg">
-              <div class="email_txt">
+              <div class="email-txt">
```

Comment:
If you have the same type of elements group please be consistent. You should use span for `email-txt` and `phone-txt"` to match the `street-txt`

Reply:
fixed - changed all for 'span'